setup.py

```diff
@@ -344,6 +344,10 @@ def run(self):
     install_requires=requires,
     cmdclass={
         'install': install
-    }
+    },
+    package_data={
+        "azurelinuxagent.common.policy.regorus": ["*.so"],
```

Comment:
I don't fully understand this executable file yet. 

Is this something that will be built in a separate repo? And then the executable .so will be added at /common/policy/regorous in our repo?

Reply:
Added some more context to the PR description - we build a Regorus wheel file from a separate repo, unzip the wheel to get the .so executable (to avoid pip), and manually add this executable to /common/policy/regorus.

Reply:
Thanks :)